### PR TITLE
Add bundled flow

### LINKED ISSUE #113: Bundling flow with the extension
Currently, the extension can either use the globally installed flow binary or the one in the `node_modules` of the project. Like the way the [prettier](https://github.com/esbenp/prettier-vscode#prettier-resolution) extension handles it. This would make setup easier. Also save unnecessary installs for simple one-off projects or files

### PR DESCRIPTION
**Pull request by BrainMaestro**
Fixes #113

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
         "cross-spawn": "^4.0.0",
         "dequeue": "^1.0.5",
         "event-kit": "^2.0.0",
+        "flow": "^0.2.3",
```
Comment (orta) [inline package.json]: I think you're looking for `flow-bin` instead - this is [flow](https://www.npmjs.com/package/flow)

**Comment by BrainMaestro** [comment]
Yea I missed that. Fixed now

**Comment by orta** [comment]
Alright - cool, this looks good - @BrainMaestro can you please add. aCHANGELOG entry to give yourself some credit (you may need to add a master section) other than that - this looks good to go 👍 

Nice work

**Comment by BrainMaestro** [comment]
@orta Thanks! I added a new section to the changelog. Is that the correct way?

**Comment by orta** [comment]
Yeah, this is great - thanks @BrainMaestro

**Pull request merged**